memory_server.py

```
@@ -25,6 +26,39 @@
+
+    Returns:
+        str: File contents or friendly error message
+    """
+    root_path = Path(current_dir) / filename
+    docs_path = Path(current_dir) / "docs" / filename
+
+    # Try project root first (for README.md, agents.md)
+    if root_path.exists():
+        try:
+            return root_path.read_text(encoding='utf-8')
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)